enarx-keep/shims/shim-sgx/src/hostlib.rs

```diff
@@ -1,34 +1,39 @@
 // SPDX-License-Identifier: Apache-2.0
 
-//! The common Layout for `enarx-keep-sgx` and `enarx-keep-sgx-shim`
-
-#![no_std]
-#![deny(clippy::all)]
-#![deny(missing_docs)]
+// The common Layout for `enarx-keep-sgx` and `enarx-keep-sgx-shim`
 
 use bounds::{Contains as _, Line, Span};
 
+#[allow(dead_code)]
```

Comment:
```suggestion
#![allow(dead_code)]
```

This would avoid a lot of redundancy.

Reply:
Actually, I've changed my mind. We shouldn't do this at all. There should be no dead code in the shared interfaces at all. That means one side isn't using it and it shouldn't be in the shared file.

Reply:
In fact, I thought I moved this code into the loader backend...

Reply:
fixed